### PR TITLE
docs: :memo: update landing page content and layout

### PR DESCRIPTION
**Pull request by signekb**
## Description

This PR updates the landing page with:

- less content (I have moved goals to their own post under "About")
- a new button linking to Sprout (to lead people to what we're currently working on)
- a new layout of the software products where it's clearer which logo goes with which app

Looks like this on my local version (with styling currently in open PRs seedcase-project/seedcase-theme#60 seedcase-project/seedcase-theme#61 seedcase-project/seedcase-theme#62): 
image

NB: The footnote is not shown here bc it didn't fit in the screenshot - but it's there)

## Reviewer Focus

This PR needs an in-depth review.

## Checklist

- [X] Ran spell-check
- [X] Formatted Markdown
- [X] Rendered website locally

### CODE REVIEW

**Review by K-Beicher — APPROVED** [review]
Have had a good nose around, and I really like it. Good work!

@lwjohnst86 do we at some point want to have a look at the About Seedcase page, the very first page talks about DD2, which we have moved away from...

**Comment by lwjohnst86** [comment]
Great idea @K-Beicher, wanna do a draft of some updates?

File: about/goals.qmd [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Specifically training on technical skills such as software and data
+engineering, while following open scientific practices.
+
+### Build and nurture an inclusive community
+
+A community that is open and inclusive that shares knowledge, learnings,
+and better practices in research data engineering and software
+engineering.
+
+### Lead by example
```
Comment (lwjohnst86) [inline about/goals.qmd:24]:
```suggestion
## Develop open-source software for health data

Software that is open-source and targeted to diverse users, like health
research organizations and small-to-medium sized companies.

## Create training material and run workshops

Specifically training on technical skills such as software and data
engineering, while following open scientific practices.

## Build and nurture an inclusive community

A community that is open and inclusive that shares knowledge, learnings,
and better practices in research data engineering and software
engineering.

## Lead by example
```

Generally not good to jump in hierarchy (title which is header 1, to header 3) and rather to keep a consistent jump (title to header 2)

**Review by lwjohnst86 — APPROVED** [review]
Only one small suggestion. Otherwise, looks nice! Not sure I love the buttons, I think the rounding is better than the square. But that isn't relevant to this PR.

File: about/goals.qmd (hunk 1 above)
Comment (signekb) [reply about/goals.qmd]: Ah, yes, that was unintentional!

**Pull request merged**